### ISSUE TITLE
New Kovan demo deployment

### ISSUE THREAD

**Issue by denniswell** [centrifuge/tinlake-deploy#22]
Problem: 
The current kovan staging deployment with NFT feed and Prop Operator is pretty much unusable for Demos and proper scenario testing (beyond end2end tests). Plus we often have conflicts with pushing new features and scheduled demos

AC:
We have one identical staging and demo deployment with two pools each. All new features would be pushed and tested on staging and only after testing merged into Mainnet/Demo. One pool is configured in line with our Mainnet deployments. The second pool has the allowance operator instead of the Prop Operator. 
TBD NFT feed.
